crates/nargo/src/cli/mod.rs

```diff
@@ -29,7 +29,7 @@ const VERIFIER_INPUT_FILE: &str = "Verifier";
 const SRC_DIR: &str = "src";
 const PKG_FILE: &str = "Nargo.toml";
 const PROOF_EXT: &str = "proof";
-const BUILD_DIR: &str = "build";
+const TARGET_DIR: &str = "check";
```

Comment:
I think you misunderstood my suggestion, it was to change the actual directory here from `"check"` to `"target"` or `"build"`, not to change the variable name again

Reply:
ohhh, got it.